alexandria/settings/alexandria.py

```diff
@@ -160,3 +160,10 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
 
 # Checksums
 ALEXANDRIA_ENABLE_CHECKSUM = env.bool("ALEXANDRIA_ENABLE_CHECKSUM", default=True)
+
+# Clamav service
+CLAMD_SOCKET = env.str("ALEXANDRIA_CLAMD_SOCKET", default="/var/run/clamav/clamd.ctl")
+CLAMD_USE_TCP = env.bool("ALEXANDRIA_CLAMD_USE_TCP", default=True)
+CLAMD_TCP_SOCKET = env.str("ALEXANDRIA_CLAMD_TCP_SOCKET", default=3310)
+CLAMD_TCP_ADDR = env.str("ALEXANDRIA_CLAMD_TCP_ADDR", default="localhost")
+CLAMD_ENABLED = env.bool("ALEXANDRIA_CLAMD_ENABLED", default=True)
```

Comment:
```suggestion
CLAMD_ENABLED = env.bool("ALEXANDRIA_CLAMD_ENABLED", default=default(False,True))
```
This would disable it in dev, but enable in prod, unless explicitly configured.

I'm also not sure I'd enable it by default at all - no strong opinions, but I feel like this shouldn't be enabled by default

Reply:
Makes sense otherwise this is breaking.
Changed default to false.